docker-compose.yml

```diff
@@ -0,0 +1,230 @@
+    build:
+      context: .
+      dockerfile: Dockerfile
+    # For production
+    #image: monogramm/docker-taiga-back:4.2-alpine
+    hostname: ${HOSTNAME}
+    container_name: taiga_back
+    #restart: always
+    depends_on:
+      - taigadb
+    ports:
+      - 8001:8001
+    volumes:
+      # Media and uploads directory. Required (or you will lose all uploads)
+      - /srv/taiga/back/media:/usr/src/taiga-back/media
+      - /srv/taiga/back/static:/usr/src/taiga-back/static
+      # Taiga configuration directory. Makes it easier to change configuration with your own
+      #- /srv/taiga/back/conf:/taiga
+      - /usr/local/lib/python3.6/site-packages/taiga_contrib_ldap_auth_ext
+      - /etc/localtime:/etc/localtime:ro
+      - /etc/timezone:/etc/timezone:ro
+    environment:
+      # Your hostname (REQUIRED)
+      - TAIGA_HOSTNAME=${HOSTNAME}:${PORT}
+      #- TAIGA_SSL=False
+      #- TAIGA_SSL_BY_REVERSE_PROXY=True
+      # Secret key for cryptographic signing
+      - TAIGA_SECRET_KEY=${TAIGA_SECRET}
+      # Admin account default password
+      - TAIGA_ADMIN_PASSWORD=${TAIGA_ADMIN_PASSWORD}
+      # Database settings
+      - TAIGA_DB_HOST=taigadb
+      - TAIGA_DB_NAME=taiga
+      - TAIGA_DB_USER=taiga
+      - TAIGA_DB_PASSWORD=${TAIGA_DB_PWD}
+      # when the db comes up from docker, it is usually too quick
+      - TAIGA_SLEEP=5
+      # To use an external SMTP for emails, fill in these values:
+      - TAIGA_ENABLE_EMAIL=True
+      - TAIGA_EMAIL_FROM=taiga@${TAIGA_EMAIL_DOMAIN}
+      - TAIGA_EMAIL_USE_TLS=True
+      - TAIGA_EMAIL_HOST=smtp.${TAIGA_EMAIL_DOMAIN}
+      - TAIGA_EMAIL_PORT=587
+      - TAIGA_EMAIL_USER=${TAIGA_SMTP_USER}
+      - TAIGA_EMAIL_PASS=${TAIGA_SMTP_PWD}
+      # Backend settings
+      - TAIGA_DEBUG=False
+      - TAIGA_PUBLIC_REGISTER_ENABLED=False
+      - TAIGA_FEEDBACK_ENABLED=True
+      - TAIGA_FEEDBACK_EMAIL=taiga@${TAIGA_EMAIL_DOMAIN}
+      # Events settings
+      - TAIGA_EVENTS_ENABLED=True
+      - RABBIT_USER=${TAIGA_RABBIT_USER}
+      - RABBIT_PASSWORD=${TAIGA_RABBIT_PASSWORD}
+      - RABBIT_VHOST='/'
+      - RABBIT_HOST=taiga_rabbit
+      - RABBIT_PORT=5672
+      # Async settings
+      # To enable async mode, uncomment the following lines:
+      #- TAIGA_ASYNC_ENABLED=True
+      #- REDIS_HOST=taiga_redis
+      #- REDIS_PORT=6379
+      ### Additional parameters
+      # LDAP Settings
+      - TAIGA_ENABLE_LDAP=${TAIGA_ENABLE_LDAP}
+      - TAIGA_LDAP_USE_TLS=False
+      - TAIGA_LDAP_SERVERS=ldaps://${TAIGA_LDAP_DOMAIN_1},ldaps://${TAIGA_LDAP_DOMAIN_2},ldaps://${TAIGA_LDAP_DOMAIN_3}
+      - TAIGA_LDAP_PORT=636
+      - TAIGA_LDAP_BIND_DN=${TAIGA_LDAP_BIND_DN}
+      - TAIGA_LDAP_BIND_PASSWORD=${TAIGA_LDAP_BIND_PASSWORD}
+      - TAIGA_LDAP_BASE_DN=${TAIGA_LDAP_BASE_DN}
+      - TAIGA_LDAP_USERNAME_ATTRIBUTE=uid
+      - TAIGA_LDAP_EMAIL_ATTRIBUTE=mail
+      - TAIGA_LDAP_FULL_NAME_ATTRIBUTE=displayName
+      - TAIGA_LDAP_FALLBACK=normal
```

Comment:
@AminovE99 Setup this to use a local [openldap](https://github.com/osixia/docker-openldap) instance.